Dockerfile-decrediton

```diff
@@ -20,7 +20,7 @@ RUN adduser --disabled-password --gecos ''  $USER
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && \
-    apt-get install -qy build-essential python git curl
+    apt-get install -qy build-essential python git curl librpmbuild3
```

Comment:
This should add the 'rpm' package, not the libs.  The libs won't contain the rpmbuild binary.

https://packages.ubuntu.com/trusty/amd64/rpm/filelist

https://packages.ubuntu.com/trusty/amd64/librpmbuild3/filelist